orbit/pkg/table/user_exec/exec.go

```diff
@@ -0,0 +1,144 @@
+// build +darwin
+
+package user_exec
```

Comment:
Needs package docs:
```go
// Package user_exec ...
package user_exec
```
(Guessing this is a generic package to allow this kind of mechanism to run osquery as user)